### DISCUSSION TITLE
How to use Grott with two inverters

### DISCUSSION THREAD

**Discussion by daath** [johanmeijer/grott#328] [General]
Hi all!

I am new to Home Assistant, Grott and MQTT so please bear with me. Grott is a great project!

I have a SPH3600 and a MIC 3000TL-X inverter, and I set up Grott and changed the dataloggers to report to grott - it works beautifully. However, I am unsure of how to read out the correct (aggregated) values (or indeed aggregate them myself somehow).
I put the basic sensor in Home Assistant:

![image](https://user-images.githubusercontent.com/543124/233208444-edc1c642-61c1-4317-acf5-a7c3a73249d5.png)

I get the feeling that 99% of people using Grott juse use one inverter, so it just works for them ...

And it does show something, just not what I would expect (not sure what I expected). It looks like the data is just from one of the inverters.

I can see in MQTT that it gets one message from one inverter with a specific serial # - then shortly after it gets data from the other with another serial # - so ... How would I go about getting the correct (aggregated) data to show?

EDIT: Is the answer to run 2 Grott instances on different ports and create a sensor that just pulls from two topics? Two instances wouldn't be a problem, since I run it as a docker instance in Portainer :)

I hope you can help a noob!
 
:)

**Comment by daath** [comment]
Two instances seemed to do the trick. Now I want to figure out how to integrate it into the Energy Dashboard...

**Comment by daath** [comment]
I must have been tired. I completely missed https://github.com/johanmeijer/grott/tree/master/examples/Home%20Assistent - so I guess I did it all wrong. No wonder I thought it was very complicated to use 🤣
I will re-do my setup when I have the energy!

**Reply by rany2** [reply]
> when I have the energy!

:rofl:
